Fix #17538 Index Rename Button generated incorrect sql

kamil-tekiela opened the pull request:
Fix #17538

kamil-tekiela committed:
Fix #17538 Index Rename Button generated incorrect sql

Signed-off-by: Kamil Tekiela <tekiela246@gmail.com>

williamdes reviewed (approved):
Thank you !
Can you add an unit test so we can ensure the query is clean now ?

kamil-tekiela commented:
> Thank you ! Can you add an unit test so we can ensure the query is clean now ?

I thought about it, but it seems to be quite a challenge. I'd rather submit the test on master branch where it would be easier to do it. But I also think that this should be redesigned on master. So I don't even know which behaviour should be tested. I did find tests that should cover this feature, but the expectations are wrong. 

Would you be able to help me with an easy unit test for 5.2?

MauricioFauth committed:
Add unit test Table\IndexRenameController class

Adds unit test for preview SQL button when renaming an index and using
an old MySQL server.

Signed-off-by: Maurício Meneghini Fauth <mauricio@fauth.dev>

MauricioFauth commented:
> Would you be able to help me with an easy unit test for 5.2?

I added an unit test that covers the path for the preview SQL action.
